docs/gateway_docs/development.rst

```diff
@@ -0,0 +1,42 @@
+Gateway and zigbee devices
+====
+
+### Supporting new push actions
+
+To support new push actions or properties a packet capture of the Xiaomi Home app needs to be made to figure out the required data.
+To do this you will need 3 programs running on a PC:
+ - [BlueStacks](https://www.bluestacks.com) to emulate the Xiaomi Home app on windows
+ - [WireShark](https://www.wireshark.org) to capture the network packets send by the Xiaomi Home app in BlueStacks
+ - [Miio Tool](https://github.com/aholstenson/miio) to decode the captured packets of WireShark
```

Comment:
@rytilahti could you detail how to use the new parse_pcap.py script?
I tried:
`python3 parse_pcap.py "C:\Users\Name\Downloads\test.pcapng" --token TokenTokenToken`
But that gave me:
```
Traceback (most recent call last):
  File "parse_pcap.py", line 15, in <module>
    def read_payloads_from_file(file, tokens: list[str]):
TypeError: 'type' object is not subscriptable
```
and saving a capture from wireshark gives me a `pcapng` file instead of a pcap file, is that fine?

Reply:
Oh, your python version is probably <3.9, as that release was the first one allowing this form instead of using `typing.List`. You could either add `from __future__ import annotations` or change those non-subscriptables to be the uppercased ones from the typing module (e.g., `from typing import List`, and using `List[str]` here).

pcapng should be fine, but if it doesn't work, you can easily convert it by using wireshark.

Reply:
@rytilahti thanks, I got it working by updating to python 3.10.
pcapng did not work, but saving it as pcap did.

I updated the instructions to use the new devtools.

Schould I move the intructions from `docs/gateway_docs/development.md` to `/devtools/gateway_development.md` ?